### DISCUSSION TITLE
dataset format for multi turn conversation with context

### DISCUSSION THREAD

**Discussion by dinchu** [axolotl-ai-cloud/axolotl#912] [Q&A]
I was checking all the supported formats but i don't see any that mixes a conversation with an instruct/context

What is the best format for a conversation that has a context? example of what i would like:

`{"instruction": "this is a conversation between two teenagers", "conversations": [{"from": "...", "value": "..."}]}`

**Comment by NanoCode012** [comment]
You may be looking to use the `system` prompt instead of instruction. Your first row should be 

```
[{"from": "system", "value": "this is a conversation between two teenagers"}]
```
